core/src/main/resources/hudson/PluginManager/advanced_bg.properties

```diff
@@ -1,6 +1,5 @@
 # This file is under the MIT License by authors
 
-Check\ now=\u041F\u0440\u043E\u0432\u0435\u0440\u0438 \u0441\u0435\u0433\u0430
```

Comment:
BTW not sure if you found it or not, but `core/move-l10n.groovy` is the recommended easy way to move bundle keys with localizations.